scripts/git_api_curl.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/sh
 
 # curl -X POST -H "Accept: application/json" -H "Authorization: Bearer $GITHUB_SECURITY_ACCESS_TOKEN" https://api.github.com/repos/$BUILDKITE_ORGANIZATION_SLUG/$BUILDKITE_PIPELINE_SLUG/commits/$BUILDKITE_COMMIT/comments -d '{"body":"A new potential secret was discovered in this commit @peakon/security"}'
-curl -X POST -H 'Content-type: application/json' --data "{\"text\":\"Potential secret in Github!\", \"attachments\":[{\"color\": \"warning\", \"title\": \"A new potential secret was commited in Github\", \"text\": \"https://github.com/$BUILDKITE_ORGANIZATION_SLUG/$BUILDKITE_PIPELINE_SLUG/commit/$BUILDKITE_COMMIT\"}]}" $SF_DEFAULT_SLACK_WEBHOOK_URL
-
+curl -X POST -H 'Content-type: application/json' --data "{\"text\":\"Potential secret in Github!\", \"attachments\":[{\"color\": \"warning\", \"title\": \"A new potential secret was commited in Github\", \"text\": \"https://github.com/$BUILDKITE_ORGANIZATION_SLUG/$BUILDKITE_PIPELINE_SLUG/commit/$BUILDKITE_COMMIT\"}]}" $PEAKBOT_WEBHOOK_SECRET
```

Comment:
I am not sure this is the right secret, this is for authenticating to peakbot

Reply:
You either need to use [this](https://github.com/peakon/ci-scripts-buildkite-plugin/blob/6450f700687c81bd7a00b1745cda91312539ad98/bin/notify-slack-channel) method to send via the peakbot API or you can get the `peakbot-slack-token` secret and use it to call the slack API [directly](https://api.slack.com/tutorials/tracks/posting-messages-with-curl)

Reply:
@threesquared Have we stored `peakbot-slack-token` in param store? I can't find it in CI.

Reply:
Ah I think maybe only ArgoCD uses it, can you copy it from the management account?

Reply:
I don't have access to add to param store

Reply:
Added as `/vendors/buildkite/deploy/global/env/PEAKBOT_SLACK_TOKEN` now

Reply:
Recent changes in [3d0bcda](https://github.com/peakon/docker-detect-secrets/pull/9/commits/3d0bcda4e1de6fbc8abcdff0b6a31d3de989df07).